telegram/ext/_dispatcher.py

```diff
@@ -631,6 +652,78 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
             if not self.handlers[group]:
                 del self.handlers[group]
 
+    def drop_chat_data(self, chat_id: int = None, all_empty_entries: bool = False) -> None:
+        """Used for deleting a key from the :attr:`chat_data`.
+
+        .. versionadded:: 14.0
+
+        Args:
+            chat_id (:obj:`int`, optional): The chat id to delete from the persistence. The entry
+                will be deleted even if it is not empty. Mutually exclusive with
+                :paramref:`all_empty_entries`.
+            all_empty_entries (:obj:`bool`, optional): If :obj:`True`, all empty entries from
+                :attr:`chat_data` will be deleted. Mutually exclusive with :paramref:`chat_id`.
+
+        Raises:
+            :exc:`ValueError`: When both :paramref:`chat_id` and :paramref:`all_empty_entries` are
```

Comment:
just wordering: how does `paramref` render while we don't have support for it?

Reply:
it doesn't. It just shows the text ``` `:paramref:`...` ```

Reply:
then I'd prefer to stick to `` for now :)